libcodechecker/analyze/analyzers/analyzer_types.py

```diff
@@ -22,6 +21,7 @@
 from libcodechecker.analyze.analyzers import result_handler_clang_tidy
 from libcodechecker.analyze.analyzers import result_handler_plist_to_db
 from libcodechecker.analyze.analyzers import result_handler_plist_to_stdout
+from libcodechecker.cmd.cmd_line_client import setup_client
```

Comment:
That's why I'd go with introducing a `TODO` at this point. There are a lot of such `TODO`s that are to be handled later on. I would like this pull request to be minimal in terms of changes related to the new storage system, for review and history's sake.

Cross-package imports are another concern, they can be refactored out in a batch.